radiography/src/main/java/radiography/internal/ComposeLayoutInfo.kt

```diff
@@ -50,18 +51,19 @@ private fun Group.computeLayoutInfos(
 
   // Look for any CompositionReferences stored in this group. These will be rolled up into the
   // SubcomposeLayout if present, otherwise they will just be shown as regular children.
+  @OptIn(InternalComposeApi::class)
   val subComposedChildren = getCompositionReferences()
-      .flatMap { it.tryGetComposers().asSequence() }
-      .map { subcomposer ->
-        ComposeLayoutInfo(
-            isSubcomposition = true,
-            name = name,
-            bounds = box,
-            modifiers = emptyList(),
-            children = subcomposer.slotTable.asTree().layoutInfos,
-            view = null
-        )
-      }
+    .flatMap { it.tryGetComposers().asSequence() }
+    .map { subcomposer ->
+      ComposeLayoutInfo(
+        isSubcomposition = true,
+        name = name,
+        bounds = box,
+        modifiers = emptyList(),
+        children = subcomposer.compositionData.asTree().layoutInfos,
```

Comment:
`slotTable` is now private, replaced by these more limited-surface-area APIs. They're still not intended for public consumption, hence the need for the `@OptIn` above.

Reply:
Worth saying so in a code comment, that `compositionData` specifically is "cheating."